account_invoice_inter_company/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
I think that maybe it is not necessary to show this information here, because company_id is not visible on the current system of `res.config.settings`. Moreover, you can configure it in company form.

Reply:
You're right, can you check now?

Reply:
Now it's great on `res.config.settings`.  I think it was useful on `res.company` form, but it can work this way also.